{"pagelen": 100, "values": [{"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36713654.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36713654"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Just after posting this issue I noticed that version of libgazebo was 7.6 on that computer. Hopefully, it is fixed in 7.7.", "markup": "markdown", "html": "<p>Just after posting this issue I noticed that version of libgazebo was 7.6 on that computer. Hopefully, it is fixed in 7.7.</p>", "type": "rendered"}, "created_on": "2017-05-11T05:22:46.856229+00:00", "user": {"display_name": "Vinayak Jagtap", "uuid": "{4ca9a5d1-e34c-4476-9994-ccb04b8c4c73}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D"}, "html": {"href": "https://bitbucket.org/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/d6b7a3e100816973cdc8ca51912fc868d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsVJ-4.png"}}, "nickname": "vinayak_jagtap", "type": "user", "account_id": "557058:e3f455ed-c5fc-439b-b24b-e6740163bb87"}, "updated_on": null, "type": "issue_comment", "id": 36713654}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36747675.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36747675"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "It fails with gazebo 7.7 also, with the same message\n`gzserver[23163]: segfault at 10 ip 00007f7181ee2b7e sp 00007f7136ffc700 error 4 in libgazebo_transport.so.7.7.0[7f7181e9d000+69000]`", "markup": "markdown", "html": "<p>It fails with gazebo 7.7 also, with the same message\n<code>gzserver[23163]: segfault at 10 ip 00007f7181ee2b7e sp 00007f7136ffc700 error 4 in libgazebo_transport.so.7.7.0[7f7181e9d000+69000]</code></p>", "type": "rendered"}, "created_on": "2017-05-11T07:39:01.892286+00:00", "user": {"display_name": "sumanth nirmal", "uuid": "{b3a1a8e9-f698-4793-bfa9-168159ce0416}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bb3a1a8e9-f698-4793-bfa9-168159ce0416%7D"}, "html": {"href": "https://bitbucket.org/%7Bb3a1a8e9-f698-4793-bfa9-168159ce0416%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/ae4ea655bd1fa64349224c2624e423b6d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSN-3.png"}}, "nickname": "Sumanth-Nirmal", "type": "user", "account_id": "557058:60351c47-1efe-43f2-80df-6f27f6457ece"}, "updated_on": null, "type": "issue_comment", "id": 36747675}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36763129.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36763129"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2017-05-11T16:50:58.454242+00:00", "user": {"display_name": "Louise Poubel", "uuid": "{5cfa2075-477b-4ded-bdb9-8d2479544ec4}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D"}, "html": {"href": "https://bitbucket.org/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8/7d903d90-c5ea-4182-b7ef-0d467e9e1c74/128"}}, "nickname": "chapulina", "type": "user", "account_id": "557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8"}, "updated_on": null, "type": "issue_comment", "id": 36763129}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36763163.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36763163"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Is this happening for all tasks? Which world are you using? What messages do you see on the terminal before the crash? For example:\n\n~~~\n[Msg] Task [1] - Started: time will start counting as you leave the box.\n[Msg] Started box contains plugin [task1/start]\n[Msg] Task [1] - Checkpoint [1] - Started (65 21000000)\n[Msg] Started box contains plugin [task1/checkpoint1]\n~~~", "markup": "markdown", "html": "<p>Is this happening for all tasks? Which world are you using? What messages do you see on the terminal before the crash? For example:</p>\n<div class=\"codehilite\"><pre><span></span>[Msg] Task [1] - Started: time will start counting as you leave the box.\n[Msg] Started box contains plugin [task1/start]\n[Msg] Task [1] - Checkpoint [1] - Started (65 21000000)\n[Msg] Started box contains plugin [task1/checkpoint1]\n</pre></div>", "type": "rendered"}, "created_on": "2017-05-11T16:53:21.883489+00:00", "user": {"display_name": "Louise Poubel", "uuid": "{5cfa2075-477b-4ded-bdb9-8d2479544ec4}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D"}, "html": {"href": "https://bitbucket.org/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8/7d903d90-c5ea-4182-b7ef-0d467e9e1c74/128"}}, "nickname": "chapulina", "type": "user", "account_id": "557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8"}, "updated_on": null, "type": "issue_comment", "id": 36763163}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36765689.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36765689"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "We were using unique_task1.world, I'll add more details when I see that error again.", "markup": "markdown", "html": "<p>We were using unique_task1.world, I'll add more details when I see that error again.</p>", "type": "rendered"}, "created_on": "2017-05-11T18:51:08.699094+00:00", "user": {"display_name": "Vinayak Jagtap", "uuid": "{4ca9a5d1-e34c-4476-9994-ccb04b8c4c73}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D"}, "html": {"href": "https://bitbucket.org/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/d6b7a3e100816973cdc8ca51912fc868d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsVJ-4.png"}}, "nickname": "vinayak_jagtap", "type": "user", "account_id": "557058:e3f455ed-c5fc-439b-b24b-e6740163bb87"}, "updated_on": null, "type": "issue_comment", "id": 36765689}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36769792.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36769792"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Don't have details to post but I've seen it, also, a couple times.", "markup": "markdown", "html": "<p>Don't have details to post but I've seen it, also, a couple times.</p>", "type": "rendered"}, "created_on": "2017-05-11T23:24:35.900778+00:00", "user": {"display_name": "Rud Merriam", "uuid": "{79efa10c-b8b6-4b55-9d05-7d0057e235cb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D"}, "html": {"href": "https://bitbucket.org/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc/5c7bde69-befb-417b-9aae-9db0c9a59777/128"}}, "nickname": "rmerriam", "type": "user", "account_id": "557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc"}, "updated_on": null, "type": "issue_comment", "id": 36769792}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36789632.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36789632"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Thanks. If you're building from source, one thing that might help when upgrading Gazebo versions is to recompile your whole stack (delete the build folder).", "markup": "markdown", "html": "<p>Thanks. If you're building from source, one thing that might help when upgrading Gazebo versions is to recompile your whole stack (delete the build folder).</p>", "type": "rendered"}, "created_on": "2017-05-12T15:30:56.957573+00:00", "user": {"display_name": "Louise Poubel", "uuid": "{5cfa2075-477b-4ded-bdb9-8d2479544ec4}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D"}, "html": {"href": "https://bitbucket.org/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8/7d903d90-c5ea-4182-b7ef-0d467e9e1c74/128"}}, "nickname": "chapulina", "type": "user", "account_id": "557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8"}, "updated_on": null, "type": "issue_comment", "id": 36789632}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36790476.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36790476"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "I have seen this happen right after the Task 1 Checkpoint 1 message is posted on the console. I'm *not* building from source, using only the packaged competition software.", "markup": "markdown", "html": "<p>I have seen this happen right after the Task 1 Checkpoint 1 message is posted on the console. I'm <em>not</em> building from source, using only the packaged competition software.</p>", "type": "rendered"}, "created_on": "2017-05-12T16:07:08.144566+00:00", "user": {"display_name": "Rud Merriam", "uuid": "{79efa10c-b8b6-4b55-9d05-7d0057e235cb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D"}, "html": {"href": "https://bitbucket.org/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc/5c7bde69-befb-417b-9aae-9db0c9a59777/128"}}, "nickname": "rmerriam", "type": "user", "account_id": "557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc"}, "updated_on": null, "type": "issue_comment", "id": 36790476}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36796165.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36796165"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "If you are running inside docker, you can pass `--ulimit core=1000000000:1000000000` to the `docker run` command so that it will generate corefiles. I think they will be created in `~/.ros/`. That could be useful for indicating what caused the seg-fault.", "markup": "markdown", "html": "<p>If you are running inside docker, you can pass <code>--ulimit core=1000000000:1000000000</code> to the <code>docker run</code> command so that it will generate corefiles. I think they will be created in <code>~/.ros/</code>. That could be useful for indicating what caused the seg-fault.</p>", "type": "rendered"}, "created_on": "2017-05-12T23:06:17.897263+00:00", "user": {"display_name": "Steve Peters", "uuid": "{2ccfed09-18b8-4921-8d58-15ef01092802}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D"}, "html": {"href": "https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1fb4816dad9e68337d3096f750951f6cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSP-1.png"}}, "nickname": "Steven Peters", "type": "user", "account_id": "557058:5de38267-b118-494c-aa76-4fab35448816"}, "updated_on": null, "type": "issue_comment", "id": 36796165}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36796898.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36796898"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "I am not running inside docker, this is the message I see on terminal after segfault:\n\n```\n[Msg] Task [1] - Started: time will start counting as you leave the box.\n\n[Msg] Started box contains plugin [task1/start]\n[Msg] Task [1] - Checkpoint [1] - Started (82 972000000)\nSegmentation fault\n[gazebo-6] process has died [pid 23258, exit code 139, cmd /opt/ros/indigo/lib/gazebo_ros/gzserver --verbose -e ode /opt/ros/indigo/share/srcsim/worlds/unique_task1.world\n```", "markup": "markdown", "html": "<p>I am not running inside docker, this is the message I see on terminal after segfault:</p>\n<div class=\"codehilite\"><pre><span></span>[Msg] Task [1] - Started: time will start counting as you leave the box.\n\n[Msg] Started box contains plugin [task1/start]\n[Msg] Task [1] - Checkpoint [1] - Started (82 972000000)\nSegmentation fault\n[gazebo-6] process has died [pid 23258, exit code 139, cmd /opt/ros/indigo/lib/gazebo_ros/gzserver --verbose -e ode /opt/ros/indigo/share/srcsim/worlds/unique_task1.world\n</pre></div>", "type": "rendered"}, "created_on": "2017-05-13T02:17:27.361554+00:00", "user": {"display_name": "sumanth nirmal", "uuid": "{b3a1a8e9-f698-4793-bfa9-168159ce0416}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bb3a1a8e9-f698-4793-bfa9-168159ce0416%7D"}, "html": {"href": "https://bitbucket.org/%7Bb3a1a8e9-f698-4793-bfa9-168159ce0416%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/ae4ea655bd1fa64349224c2624e423b6d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSN-3.png"}}, "nickname": "Sumanth-Nirmal", "type": "user", "account_id": "557058:60351c47-1efe-43f2-80df-6f27f6457ece"}, "updated_on": null, "type": "issue_comment", "id": 36796898}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/36873731.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-36873731"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Seeing the same error in task 2 when checkpoint 1 is completed. Are there any specific logs that we should share from local computer not running docker that would help in troubleshooting this issue?\n\nsrcsim is installed using apt-get.\n\n\n```\n[Msg] Model [solar_panel] started touching [valkyrie] at 482 897000000 seconds\n[Msg] Model [solar_panel] touched [valkyrie] exclusively for 0 500000000 seconds\n[Msg] Stopped touch plugin [task2/checkpoint1]\n[Msg] Task [2] - Checkpoint [1] - Completed (483 399000000)\n[Msg] Task [2] - Checkpoint [2] - Started (483 399000000)\nSegmentation fault\n[gazebo-6] process has died [pid 27042, exit code 139, cmd /opt/ros/indigo/lib/gazebo_ros/gzserver --verbose -e ode /opt/ros/indigo/share/srcsim/worlds/unique_task2.world __name:=gazebo __log:=/home/whrl/.ros/log/685d873a-3a64-11e7-b645-902b34d61a86/gazebo-6.log].\nlog file: /home/whrl/.ros/log/685d873a-3a64-11e7-b645-902b34d61a86/gazebo-6*.log\n\n```", "markup": "markdown", "html": "<p>Seeing the same error in task 2 when checkpoint 1 is completed. Are there any specific logs that we should share from local computer not running docker that would help in troubleshooting this issue?</p>\n<p>srcsim is installed using apt-get.</p>\n<div class=\"codehilite\"><pre><span></span>[Msg] Model [solar_panel] started touching [valkyrie] at 482 897000000 seconds\n[Msg] Model [solar_panel] touched [valkyrie] exclusively for 0 500000000 seconds\n[Msg] Stopped touch plugin [task2/checkpoint1]\n[Msg] Task [2] - Checkpoint [1] - Completed (483 399000000)\n[Msg] Task [2] - Checkpoint [2] - Started (483 399000000)\nSegmentation fault\n[gazebo-6] process has died [pid 27042, exit code 139, cmd /opt/ros/indigo/lib/gazebo_ros/gzserver --verbose -e ode /opt/ros/indigo/share/srcsim/worlds/unique_task2.world __name:=gazebo __log:=/home/whrl/.ros/log/685d873a-3a64-11e7-b645-902b34d61a86/gazebo-6.log].\nlog file: /home/whrl/.ros/log/685d873a-3a64-11e7-b645-902b34d61a86/gazebo-6*.log\n</pre></div>", "type": "rendered"}, "created_on": "2017-05-16T19:15:40.269912+00:00", "user": {"display_name": "Vinayak Jagtap", "uuid": "{4ca9a5d1-e34c-4476-9994-ccb04b8c4c73}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D"}, "html": {"href": "https://bitbucket.org/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/d6b7a3e100816973cdc8ca51912fc868d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsVJ-4.png"}}, "nickname": "vinayak_jagtap", "type": "user", "account_id": "557058:e3f455ed-c5fc-439b-b24b-e6740163bb87"}, "updated_on": "2017-05-16T19:16:24.311225+00:00", "type": "issue_comment", "id": 36873731}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37099712.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37099712"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Saw this error twice today after picking up the solar panel in task2. I'm using `higher_array` branch.\n\n```\n[Msg] Stopped touch plugin [task2/checkpoint1]\n[Msg] Task [2] - Checkpoint [1] - Completed (113 869000000)\n[Msg] Task [2] - Checkpoint [2] - Started (113 869000000)\ngzserver: /usr/include/boost/smart_ptr/shared_ptr.hpp:653: typename boost::detail::sp_member_access<T>::type boost::shared_ptr<T>::operator->() const [with T = gazebo::transport::Publisher; typename boost::detail::sp_member_access<T>::type = gazebo::transport::Publisher*]: Assertion `px != 0' failed.\nAborted (core dumped)\n[gazebo-6] process has died [pid 2619, exit code 134, cmd /opt/ros/indigo/lib/gazebo_ros/gzserver -r --verbose -e ode /home/ninja/indigo_ws/src/space_robotics_challenge/external/srcsim/worlds/unique_task2.world __name:=gazebo __log:=/home/ninja/.ros/log/c3d2c4d0-400f-11e7-933f-94de806528fb/gazebo-6.log].\nlog file: /home/ninja/.ros/log/c3d2c4d0-400f-11e7-933f-94de806528fb/gazebo-6*.log\n```", "markup": "markdown", "html": "<p>Saw this error twice today after picking up the solar panel in task2. I'm using <code>higher_array</code> branch.</p>\n<div class=\"codehilite\"><pre><span></span><span class=\"k\">[Msg] Stopped touch plugin [task2/checkpoint1]</span>\n<span class=\"na\">[Msg] Task [2] - Checkpoint [1] - Completed (113 869000000)</span>\n<span class=\"na\">[Msg] Task [2] - Checkpoint [2] - Started (113 869000000)</span>\n<span class=\"na\">gzserver: /usr/include/boost/smart_ptr/shared_ptr.hpp:653: typename boost::detail::sp_member_access&lt;T&gt;::type boost::shared_ptr&lt;T&gt;::operator-&gt;() const [with T</span> <span class=\"o\">=</span> <span class=\"s\">gazebo::transport::Publisher; typename boost::detail::sp_member_access&lt;T&gt;::type = gazebo::transport::Publisher*]: Assertion `px != 0&#39; failed.</span>\n<span class=\"na\">Aborted (core dumped)</span>\n<span class=\"na\">[gazebo-6] process has died [pid 2619, exit code 134, cmd /opt/ros/indigo/lib/gazebo_ros/gzserver -r --verbose -e ode /home/ninja/indigo_ws/src/space_robotics_challenge/external/srcsim/worlds/unique_task2.world __name:</span><span class=\"o\">=</span><span class=\"s\">gazebo __log:=/home/ninja/.ros/log/c3d2c4d0-400f-11e7-933f-94de806528fb/gazebo-6.log].</span>\n<span class=\"na\">log file: /home/ninja/.ros/log/c3d2c4d0-400f-11e7-933f-94de806528fb/gazebo-6*.log</span>\n</pre></div>", "type": "rendered"}, "created_on": "2017-05-23T23:50:30.515365+00:00", "user": {"display_name": "Vinayak Jagtap", "uuid": "{4ca9a5d1-e34c-4476-9994-ccb04b8c4c73}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D"}, "html": {"href": "https://bitbucket.org/%7B4ca9a5d1-e34c-4476-9994-ccb04b8c4c73%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/d6b7a3e100816973cdc8ca51912fc868d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsVJ-4.png"}}, "nickname": "vinayak_jagtap", "type": "user", "account_id": "557058:e3f455ed-c5fc-439b-b24b-e6740163bb87"}, "updated_on": null, "type": "issue_comment", "id": 37099712}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37099787.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37099787"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "I suspect these Gazebo transport errors are due to race conditions, which is really tough to reproduce and debug. If anyone is running the competition through gdb, it would be great to get a backtrace.", "markup": "markdown", "html": "<p>I suspect these Gazebo transport errors are due to race conditions, which is really tough to reproduce and debug. If anyone is running the competition through gdb, it would be great to get a backtrace.</p>", "type": "rendered"}, "created_on": "2017-05-24T00:02:56.170294+00:00", "user": {"display_name": "Louise Poubel", "uuid": "{5cfa2075-477b-4ded-bdb9-8d2479544ec4}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D"}, "html": {"href": "https://bitbucket.org/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8/7d903d90-c5ea-4182-b7ef-0d467e9e1c74/128"}}, "nickname": "chapulina", "type": "user", "account_id": "557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8"}, "updated_on": null, "type": "issue_comment", "id": 37099787}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37387486.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37387486"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Saw this error while starting task2.\n```\n(gdb) bt\n#0  __GI___pthread_mutex_lock (mutex=0x0) at ../nptl/pthread_mutex_lock.c:66\n#1  0x00007fff42392694 in gazebo::Task::Update(gazebo::common::Time const&) ()\n   from /opt/ros/indigo/lib/libFinalsPlugin.so\n#2  0x00007fff42384999 in gazebo::FinalsPlugin::OnUpdate(gazebo::common::UpdateInfo const&) ()\n   from /opt/ros/indigo/lib/libFinalsPlugin.so\n#3  0x00007ffff62e9219 in ?? () from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#4  0x00007ffff62d5dc1 in gazebo::physics::World::Update() ()\n   from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#5  0x00007ffff62e400b in gazebo::physics::World::Step() () from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#6  0x00007ffff62e447d in gazebo::physics::World::RunLoop() ()\n   from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#7  0x00007ffff42c3a4a in ?? () from /usr/lib/x86_64-linux-gnu/libboost_thread.so.1.54.0\n#8  0x00007ffff55a5184 in start_thread (arg=0x7fff437fe700) at pthread_create.c:312\n#9  0x00007ffff6973bed in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111\n(gdb) \n\n```", "markup": "markdown", "html": "<p>Saw this error while starting task2.</p>\n<div class=\"codehilite\"><pre><span></span>(gdb) bt\n#0  __GI___pthread_mutex_lock (mutex=0x0) at ../nptl/pthread_mutex_lock.c:66\n#1  0x00007fff42392694 in gazebo::Task::Update(gazebo::common::Time const&amp;) ()\n   from /opt/ros/indigo/lib/libFinalsPlugin.so\n#2  0x00007fff42384999 in gazebo::FinalsPlugin::OnUpdate(gazebo::common::UpdateInfo const&amp;) ()\n   from /opt/ros/indigo/lib/libFinalsPlugin.so\n#3  0x00007ffff62e9219 in ?? () from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#4  0x00007ffff62d5dc1 in gazebo::physics::World::Update() ()\n   from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#5  0x00007ffff62e400b in gazebo::physics::World::Step() () from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#6  0x00007ffff62e447d in gazebo::physics::World::RunLoop() ()\n   from /usr/lib/x86_64-linux-gnu/libgazebo_physics.so.7\n#7  0x00007ffff42c3a4a in ?? () from /usr/lib/x86_64-linux-gnu/libboost_thread.so.1.54.0\n#8  0x00007ffff55a5184 in start_thread (arg=0x7fff437fe700) at pthread_create.c:312\n#9  0x00007ffff6973bed in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111\n(gdb) \n</pre></div>", "type": "rendered"}, "created_on": "2017-06-05T22:48:12.560650+00:00", "user": {"display_name": "Shlok Agarwal", "uuid": "{a75e6f0e-7f28-48af-99ff-6c0df5c05d5b}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Ba75e6f0e-7f28-48af-99ff-6c0df5c05d5b%7D"}, "html": {"href": "https://bitbucket.org/%7Ba75e6f0e-7f28-48af-99ff-6c0df5c05d5b%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/b19701717b54b9cb87d0318994951ea0d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSA-2.png"}}, "nickname": "shlokagarwal", "type": "user", "account_id": "557058:219d8fd2-8961-4c37-8d7d-2b613ff05e25"}, "updated_on": null, "type": "issue_comment", "id": 37387486}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37388286.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37388286"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Thanks for the backtrace. This is pointing towards something in the [Task::Update](#!/osrf/srcsim/src/6db6b8f83506d358b5b10784db022e8c0c516309/src/Task.cc?at=default&fileviewer=file-view-default#Task.cc-129) function.", "markup": "markdown", "html": "<p>Thanks for the backtrace. This is pointing towards something in the <a data-is-external-link=\"true\" href=\"#!/osrf/srcsim/src/6db6b8f83506d358b5b10784db022e8c0c516309/src/Task.cc?at=default&amp;fileviewer=file-view-default#Task.cc-129\" rel=\"nofollow\">Task::Update</a> function.</p>", "type": "rendered"}, "created_on": "2017-06-06T00:36:56.780020+00:00", "user": {"display_name": "Steve Peters", "uuid": "{2ccfed09-18b8-4921-8d58-15ef01092802}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D"}, "html": {"href": "https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1fb4816dad9e68337d3096f750951f6cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSP-1.png"}}, "nickname": "Steven Peters", "type": "user", "account_id": "557058:5de38267-b118-494c-aa76-4fab35448816"}, "updated_on": null, "type": "issue_comment", "id": 37388286}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37388331.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37388331"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "Can you provide console output and a multi-thread backtrace next time?\n\n~~~\n(gdb) thread apply all bt\n~~~\n\nThe line you've pointed at just looks like it's waiting for the mutex to be freed. Does the simulation crash or is it just hanging at this point?", "markup": "markdown", "html": "<p>Can you provide console output and a multi-thread backtrace next time?</p>\n<div class=\"codehilite\"><pre><span></span>(gdb) thread apply all bt\n</pre></div>\n\n\n<p>The line you've pointed at just looks like it's waiting for the mutex to be freed. Does the simulation crash or is it just hanging at this point?</p>", "type": "rendered"}, "created_on": "2017-06-06T00:42:52.413246+00:00", "user": {"display_name": "Steve Peters", "uuid": "{2ccfed09-18b8-4921-8d58-15ef01092802}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D"}, "html": {"href": "https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1fb4816dad9e68337d3096f750951f6cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSP-1.png"}}, "nickname": "Steven Peters", "type": "user", "account_id": "557058:5de38267-b118-494c-aa76-4fab35448816"}, "updated_on": null, "type": "issue_comment", "id": 37388331}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37447852.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37447852"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "@stevenpeter91 . I came across the same error again (while starting task2 ) and this is the console output after ``` (gdb) thread apply all bt ```\n\n\n```\nMsg] [Harness] Detached\n\nProgram received signal SIGSEGV, Segmentation fault.\n[Switching to Thread 0x7fff437fe700 (LWP 30386)]\n__GI___pthread_mutex_lock (mutex=0x0) at ../nptl/pthread_mutex_lock.c:66\n66\t../nptl/pthread_mutex_lock.c: No such file or directory.\n(gdb) thread apply all bt\n\nThread 63 (Thread 0x7ffef97fa700 (LWP 31071)):\n#0  pthread_cond_timedwait@@GLIBC_2.3.2 () at ../nptl/sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S:238\n#1  0x00007fffd4912d75 in bool boost::condition_variable::timed_wait<boost::date_time::subsecond_duration<boost::posix_time::time_duration, 1000000l> >(boost::unique_lock<boost::mutex>&, boost::date_time::subsecond_duration<boost::posix_time::time_duration, 1000000l> const&) () from /opt/ros/indigo/lib/libroscpp.so\n#2  0x00007fffd4910d8d in ros::CallbackQueue::callAvailable(ros::WallDuration) ()\n   from /opt/ros/indigo/lib/libroscpp.so\n#3  0x00007fff085412d2 in gazebo::SRCSimRosHarness::QueueThread() ()\n   from /opt/ros/indigo/lib/libsrcsim_ros_harness.so\n#4  0x00007ffff42c3a4a in ?? () from /usr/lib/x86_64-linux-gnu/libboost_thread.so.1.54.0\n#5  0x00007ffff55a5184 in start_thread (arg=0x7ffef97fa700) at pthread_create.c:312\n#6  0x00007ffff6973bed in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111\n\nThread 62 (Thread 0x7ffef9ffb700 (LWP 31060)):\n#0  pthread_cond_timedwait@@GLIBC_2.3.2 () at ../nptl/sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S:238\n#1  0x00007fffd4912d75 in bool boost::condition_variable::timed_wait<boost::date_time::subsecond_duration<boost::posix_time::time_duration, 1000000l> >(boost::unique_lock<boost::mutex>&, boost::date_time::subsecond_duration<boost::posix_time::time_duration, 1000000l> const&) () from /opt/ros/indigo/lib/libroscpp.so\n#2  0x00007fffd4910d8d in ros::CallbackQueue::callAvailable(ros::WallDuration) ()\n   from /opt/ros/indigo/lib/libroscpp.so\n#3  0x00007fff0af77d2a in gazebo::GazeboRosIMU::IMUQueueThread() () from /opt/ros/indigo/lib/libgazebo_ros_imu.so\n#4  0x00007ffff42c3a4a in ?? () from /usr/lib/x86_64-linux-gnu/libboost_thread.so.1.54.0\n#5  0x00007ffff55a5184 in start_thread (arg=0x7ffef9ffb700) at pthread_create.c:312\n#6  0x00007ffff6973bed in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111\n\nThread 61 (Thread 0x7ffefa7fc700 (LWP 31049)):\n#0  pthread_cond_timedwait@@GLIBC_2.3.2 () at ../nptl/sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S:238\n```\n\nThe controller crashed after this.", "markup": "markdown", "html": "<p>@stevenpeter91 . I came across the same error again (while starting task2 ) and this is the console output after <code>(gdb) thread apply all bt</code></p>\n<div class=\"codehilite\"><pre><span></span>Msg] [Harness] Detached\n\nProgram received signal SIGSEGV, Segmentation fault.\n[Switching to Thread 0x7fff437fe700 (LWP 30386)]\n__GI___pthread_mutex_lock (mutex=0x0) at ../nptl/pthread_mutex_lock.c:66\n66  ../nptl/pthread_mutex_lock.c: No such file or directory.\n(gdb) thread apply all bt\n\nThread 63 (Thread 0x7ffef97fa700 (LWP 31071)):\n#0  pthread_cond_timedwait@@GLIBC_2.3.2 () at ../nptl/sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S:238\n#1  0x00007fffd4912d75 in bool boost::condition_variable::timed_wait&lt;boost::date_time::subsecond_duration&lt;boost::posix_time::time_duration, 1000000l&gt; &gt;(boost::unique_lock&lt;boost::mutex&gt;&amp;, boost::date_time::subsecond_duration&lt;boost::posix_time::time_duration, 1000000l&gt; const&amp;) () from /opt/ros/indigo/lib/libroscpp.so\n#2  0x00007fffd4910d8d in ros::CallbackQueue::callAvailable(ros::WallDuration) ()\n   from /opt/ros/indigo/lib/libroscpp.so\n#3  0x00007fff085412d2 in gazebo::SRCSimRosHarness::QueueThread() ()\n   from /opt/ros/indigo/lib/libsrcsim_ros_harness.so\n#4  0x00007ffff42c3a4a in ?? () from /usr/lib/x86_64-linux-gnu/libboost_thread.so.1.54.0\n#5  0x00007ffff55a5184 in start_thread (arg=0x7ffef97fa700) at pthread_create.c:312\n#6  0x00007ffff6973bed in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111\n\nThread 62 (Thread 0x7ffef9ffb700 (LWP 31060)):\n#0  pthread_cond_timedwait@@GLIBC_2.3.2 () at ../nptl/sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S:238\n#1  0x00007fffd4912d75 in bool boost::condition_variable::timed_wait&lt;boost::date_time::subsecond_duration&lt;boost::posix_time::time_duration, 1000000l&gt; &gt;(boost::unique_lock&lt;boost::mutex&gt;&amp;, boost::date_time::subsecond_duration&lt;boost::posix_time::time_duration, 1000000l&gt; const&amp;) () from /opt/ros/indigo/lib/libroscpp.so\n#2  0x00007fffd4910d8d in ros::CallbackQueue::callAvailable(ros::WallDuration) ()\n   from /opt/ros/indigo/lib/libroscpp.so\n#3  0x00007fff0af77d2a in gazebo::GazeboRosIMU::IMUQueueThread() () from /opt/ros/indigo/lib/libgazebo_ros_imu.so\n#4  0x00007ffff42c3a4a in ?? () from /usr/lib/x86_64-linux-gnu/libboost_thread.so.1.54.0\n#5  0x00007ffff55a5184 in start_thread (arg=0x7ffef9ffb700) at pthread_create.c:312\n#6  0x00007ffff6973bed in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:111\n\nThread 61 (Thread 0x7ffefa7fc700 (LWP 31049)):\n#0  pthread_cond_timedwait@@GLIBC_2.3.2 () at ../nptl/sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S:238\n</pre></div>\n\n\n<p>The controller crashed after this.</p>", "type": "rendered"}, "created_on": "2017-06-08T05:11:03.386380+00:00", "user": {"display_name": "Shlok Agarwal", "uuid": "{a75e6f0e-7f28-48af-99ff-6c0df5c05d5b}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Ba75e6f0e-7f28-48af-99ff-6c0df5c05d5b%7D"}, "html": {"href": "https://bitbucket.org/%7Ba75e6f0e-7f28-48af-99ff-6c0df5c05d5b%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/b19701717b54b9cb87d0318994951ea0d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSA-2.png"}}, "nickname": "shlokagarwal", "type": "user", "account_id": "557058:219d8fd2-8961-4c37-8d7d-2b613ff05e25"}, "updated_on": "2017-06-08T20:03:57.456511+00:00", "type": "issue_comment", "id": 37447852}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37496454.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37496454"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "We are seeing this occur with more frequency; I'm running on r744, and 20% of our recent 'batch' runs exhibited this.  (I'm trying to modify the batch to auto generate a thread apply bt all, but it's tricky).  My team mates report similar increases of occurrence of this.  It could be something in the patches from 0.6 to 0.7 increased the likely hood of this.  Could just be superstition.  (Timing bugs are *so* much fun!)", "markup": "markdown", "html": "<p>We are seeing this occur with more frequency; I'm running on r744, and 20% of our recent 'batch' runs exhibited this.  (I'm trying to modify the batch to auto generate a thread apply bt all, but it's tricky).  My team mates report similar increases of occurrence of this.  It could be something in the patches from 0.6 to 0.7 increased the likely hood of this.  Could just be superstition.  (Timing bugs are <em>so</em> much fun!)</p>", "type": "rendered"}, "created_on": "2017-06-10T00:31:10.774825+00:00", "user": {"display_name": "Jeremy White", "uuid": "{abab48b0-35b1-4102-8373-e88a492c5294}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D"}, "html": {"href": "https://bitbucket.org/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/2764d3b5adf560de71f57e48a30550bad=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJW-3.png"}}, "nickname": "knitfoo", "type": "user", "account_id": "557058:5de786d6-615c-44ef-8e12-f079efa38357"}, "updated_on": null, "type": "issue_comment", "id": 37496454}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37496603.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37496603"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "In contrast I'm seeing fewer problems. Have you updated Gazebo? A new package dropped last night...I think it was last night. Was up 24 hours and just go up from 4 hour hap.", "markup": "markdown", "html": "<p>In contrast I'm seeing fewer problems. Have you updated Gazebo? A new package dropped last night...I think it was last night. Was up 24 hours and just go up from 4 hour hap.</p>", "type": "rendered"}, "created_on": "2017-06-10T01:08:03.670960+00:00", "user": {"display_name": "Rud Merriam", "uuid": "{79efa10c-b8b6-4b55-9d05-7d0057e235cb}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D"}, "html": {"href": "https://bitbucket.org/%7B79efa10c-b8b6-4b55-9d05-7d0057e235cb%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc/5c7bde69-befb-417b-9aae-9db0c9a59777/128"}}, "nickname": "rmerriam", "type": "user", "account_id": "557058:c6ed1bc6-3112-49ac-9453-b3846fe562dc"}, "updated_on": null, "type": "issue_comment", "id": 37496603}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37496617.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37496617"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "ah, interesting.  Did not know about 7.8.1.1; was on 7.8.0.1.  Will try that and see.  Thanks for the tip!", "markup": "markdown", "html": "<p>ah, interesting.  Did not know about 7.8.1.1; was on 7.8.0.1.  Will try that and see.  Thanks for the tip!</p>", "type": "rendered"}, "created_on": "2017-06-10T01:11:42.229036+00:00", "user": {"display_name": "Jeremy White", "uuid": "{abab48b0-35b1-4102-8373-e88a492c5294}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D"}, "html": {"href": "https://bitbucket.org/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/2764d3b5adf560de71f57e48a30550bad=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJW-3.png"}}, "nickname": "knitfoo", "type": "user", "account_id": "557058:5de786d6-615c-44ef-8e12-f079efa38357"}, "updated_on": null, "type": "issue_comment", "id": 37496617}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37513029.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37513029"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "7.8.1.1 did seem to improve stability.  But I did get a run of gazebo crashes.  This was on a random world, using  r744 with r749 (harness) patched in.  The crash came when commanding the sim to start T3CP6.  I got a few in a row; attaching one of the traces in the off chance it helps.  I'll shift to a non random world and see if I can reproduce.", "markup": "markdown", "html": "<p>7.8.1.1 did seem to improve stability.  But I did get a run of gazebo crashes.  This was on a random world, using  r744 with r749 (harness) patched in.  The crash came when commanding the sim to start T3CP6.  I got a few in a row; attaching one of the traces in the off chance it helps.  I'll shift to a non random world and see if I can reproduce.</p>", "type": "rendered"}, "created_on": "2017-06-11T21:41:02.014617+00:00", "user": {"display_name": "Jeremy White", "uuid": "{abab48b0-35b1-4102-8373-e88a492c5294}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D"}, "html": {"href": "https://bitbucket.org/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/2764d3b5adf560de71f57e48a30550bad=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJW-3.png"}}, "nickname": "knitfoo", "type": "user", "account_id": "557058:5de786d6-615c-44ef-8e12-f079efa38357"}, "updated_on": null, "type": "issue_comment", "id": 37513029}, {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162/comments/37513033.json"}, "html": {"href": "#!/osrf/srcsim/issues/162#comment-37513033"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/162.json"}}, "type": "issue", "id": 162, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "Segfault in gazebo when robot moves out of start box"}, "content": {"raw": "", "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2017-06-11T21:41:14.125641+00:00", "user": {"display_name": "Jeremy White", "uuid": "{abab48b0-35b1-4102-8373-e88a492c5294}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D"}, "html": {"href": "https://bitbucket.org/%7Babab48b0-35b1-4102-8373-e88a492c5294%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/2764d3b5adf560de71f57e48a30550bad=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJW-3.png"}}, "nickname": "knitfoo", "type": "user", "account_id": "557058:5de786d6-615c-44ef-8e12-f079efa38357"}, "updated_on": null, "type": "issue_comment", "id": 37513033}], "page": 1, "size": 22}